debian/kolibri-server.init

```diff
@@ -94,19 +93,20 @@ do_check_nginx()
   service nginx status 1>/dev/null 2>&1
   retval=$?
   if [ $retval -ne 0  ]; then
-    service kolibri restart
+    su $KOLIBRI_USER -c "$KOLIBRI_COMMAND restart &" 
     service nginx start
   else
     service nginx reload
   fi
 }
 
 do_stop() {
-  service kolibri stop
+  su $KOLIBRI_USER -c "$KOLIBRI_COMMAND stop &"
```

Comment:
Is this risky? For instance, if a system is shutting down, could it cause the process to be killed prematurely?

Reply:
Assuming this is risky: https://github.com/learningequality/kolibri-server/pull/28

Reply:
no, it's not risky (at least not in my tests nor my logic). For #28 the problem is kolibri not stopping. This should not happen here becaue the command is executed (in the background, but executed).
And when the machine is restarted, waiting for zeroconf to finish its shutdown is not needed.

I tested it in different machines (including RPi) and worked properly but don't take it as a warranty, if you can test it more, much better. The rationale for the `&` was to avoid restarting a machine is locked for more than one minute waiting for zeroconf.  I think the tests are worth if we can include this change.